### ISSUE TITLE
Asks for .tbi, expects .csi

### ISSUE THREAD

**Issue by mferreiradasilva** [weizhouUMICH/SAIGE#315]
The SPA function argument ```vfFileIndex``` asks for a .tbi tabix index file, but as it is provided it gives an error because it's expecting a .csi file:

```
if(!grepl("\\.sav$", vcfFile) && !file.exists(paste(vcfFile,".csi", sep=""))){
      stop("ERROR! vcfFileIndex ", paste(vcfFile,".csi", sep=""), " does not exist\n")
    }
```

**Comment by d-mejia** [comment]
I got the same error and when I used a .csi file instead, the code ran.

**Comment by mferreiradasilva** [comment]
Yeah. It works fine with a .csi, but the info and manual are wrong because they specifically mention a tabix tbi.

**Comment by rbutleriii** [comment]
Notably, it used to work with a .tbi, but now it does not.

**Comment by medinacarolina** [comment]
Hi, I am running into the same problem I think is something that would be handy to be solved

**Comment by weizhouUMICH** [comment]
We have just released a new version 1.0.0. It has substantial computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests and clearer log output. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
